nuget/engine/nunit.engine.nuspec

```diff
@@ -45,20 +48,27 @@
     <file src="bin/net20/nunit.engine.core.dll" target="lib/net20" />
     <file src="bin/net20/nunit.engine.api.dll" target="lib/net20" />
     <file src="bin/net20/Mono.Cecil.dll" target="lib/net20" />
-    <file src="bin/net20/nunit-agent.exe" target="lib/net20" />
-    <file src="bin/net20/nunit-agent.exe.config" target="lib/net20" />
-    <file src="bin/net20/nunit-agent-x86.exe" target="lib/net20" />
-    <file src="bin/net20/nunit-agent-x86.exe.config" target="lib/net20" />
-    <file src="../../nuget/engine/nunit.nuget.addins" target="lib/net20"/>
+    <file src="bin/net20/nunit-agent.exe" target="content/any/net20" />
+    <file src="bin/net20/nunit-agent.exe" target="contentFiles/any/net20" />
+    <file src="bin/net20/nunit-agent.exe.config" target="content/any/net20" />
+    <file src="bin/net20/nunit-agent.exe.config" target="contentFiles/any/net20" />
+    <file src="bin/net20/nunit-agent-x86.exe" target="content/any/net20" />
+    <file src="bin/net20/nunit-agent-x86.exe" target="contentFiles/any/net20" />
+    <file src="bin/net20/nunit-agent-x86.exe.config" target="content/any/net20" />
+    <file src="bin/net20/nunit-agent-x86.exe.config" target="contentFiles/any/net20" />
+    <file src="../../nuget/engine/nunit.nuget.addins" target="content/any/net20"/>
+    <file src="../../nuget/engine/nunit.nuget.addins" target="contentFiles/any/net20"/>
     <file src="bin/netstandard1.6/nunit.engine.dll" target="lib/netstandard1.6" />
     <file src="bin/netstandard1.6/nunit.engine.core.dll" target="lib/netstandard1.6" />
     <file src="bin/netstandard1.6/nunit.engine.api.dll" target="lib/netstandard1.6" />
     <file src="bin/netstandard1.6/Mono.Cecil.dll" target="lib/netstandard1.6" />
-    <file src="../../nuget/engine/nunit.nuget.addins" target="lib/netstandard1.6"/>
+    <file src="../../nuget/engine/nunit.nuget.addins" target="content/any/netstandard1.6"/>
+    <file src="../../nuget/engine/nunit.nuget.addins" target="contentFiles/any/netstandard1.6"/>
     <file src="bin/netstandard2.0/nunit.engine.dll" target="lib/netstandard2.0" />
     <file src="bin/netstandard2.0/nunit.engine.core.dll" target="lib/netstandard2.0" />
     <file src="bin/netstandard2.0/nunit.engine.api.dll" target="lib/netstandard2.0" />
     <file src="bin/netstandard2.0/Mono.Cecil.dll" target="lib/netstandard2.0" />
-    <file src="../../nuget/engine/nunit.nuget.addins" target="lib/netstandard2.0"/>
+    <file src="../../nuget/engine/nunit.nuget.addins" target="content/any/netstandard2.0"/>
+    <file src="../../nuget/engine/nunit.nuget.addins" target="contentFiles/any/netstandard2.0"/>
```

Comment:
The addins file isn't moving because it's following the agent but rather because only assemblies to reference belong in the lib folder.